### DISCUSSION TITLE
List of planned games

### DISCUSSION THREAD

**Discussion by Artonix14** [mmdanggg2/D3D9DrvRTX#11] [General]
Do you have any list of planned games to support ? Will there be any other HP games ?

**Comment by mmdanggg2** [comment]
Currently working on Harry Potter 2 Chamber of Secrets, so that's next.
As for what's after that, not decided yet. There's not all that many: https://en.wikipedia.org/wiki/Category:Unreal_Engine_1_games

**Reply by Shrewtl** [reply]
Sounds good can’t wait!

**Comment by gedas919** [comment]
I really wanted Mobile Forces.
